confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/StorageGroupPartitionTable.java

```diff
@@ -207,17 +205,14 @@ public TConsensusGroupId getPrecededDataPartition(
    */
   public void createSchemaPartition(SchemaPartitionTable assignedSchemaPartition) {
     // Cache assigned result
-    Map<TConsensusGroupId, AtomicInteger> deltaMap =
+    // Map<TConsensusGroupId, Map<TSeriesPartitionSlot, deltaTimeSlotCount>>
+    Map<TConsensusGroupId, Map<TSeriesPartitionSlot, AtomicLong>> groupDeltaMap =
         schemaPartitionTable.createSchemaPartition(assignedSchemaPartition);
 
-    // Add counter
-    AtomicInteger total = new AtomicInteger(0);
-    deltaMap.forEach(
-        ((consensusGroupId, delta) -> {
-          total.getAndAdd(delta.get());
-          regionGroupMap.get(consensusGroupId).addCounter(delta.get());
-        }));
-    seriesPartitionSlotsCount.getAndAdd(total.get());
+    // Update counter
+    groupDeltaMap.forEach(
+        ((consensusGroupId, deltaMap) ->
```

Comment:
deltaMap, Map<TSeriesPartitionSlot, AtomicLong>, AtomicLong value is always 0, do we need execute the `AtomicLong` method?

Reply:
For SchameRegionGroup, the AtomicLong value is always 0. For DataRegionGroup, the AtomicLong value represents the TimeSlotCount of the specified TSeriesPartitionSlot. Currently I mingle the SchemaRegionGroup and the DataRegionGroup into the RegionGroup, for which might cause some misunderstands. I'll optimize these code in the future~